input/cql/BoneDensityProstateCancerAndrogenDeprivationTherapyFHIR.cql

```diff
@@ -142,6 +153,9 @@ define "Has Qualifying Encounter":
         and OfficeVisit.status = 'finished'
   )
 
+/*
+@QICore6Uplift: Update Condition to ConditionProblemsHealthConcerns
+*/    
```

Comment:
As written this should also be a union of ConditionProblemsHealthConcerns and ConditionEncounterDiagnosis